styles/pages/Home.module.scss

```diff
@@ -84,6 +85,9 @@
     margin-left: 10rem;
 
     .details {
+      background: linear-gradient(rgba(255, 255, 255, 0.6), rgb(255, 255, 255)),
```

Comment:
Can we extend the image to cover the entire width of the screen? Currently it is only in the center.

![image](https://user-images.githubusercontent.com/50544486/139595301-8550f551-bcbc-45c7-af4a-562ed9598130.png)

Reply:
Fixed White gap bug in commit 6dbaa04758abfb94ed768e2515a709435756282e
Fixed image in commmit f3a42128f5e13b676233930f5813fda992f59498